docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -117,7 +82,7 @@ Select "F#: New Project" by hitting **Enter**, which will take you to this step:
 
 ![](media/getting-started-vscode/vscode-proj-type.png)
 
-This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](https://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
+This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](http://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
```

Comment:
Why did you change the URL to HTTP here, when http://fslab.org redirects to https://fslab.org? I think it makes sense to use HTTPS URLs by default for sites where HTTPS works.

Reply:
Ah, this is fun. So I cloned this repo before @mairaw's PR to change all links to https, so it now registers as a change.